### PR TITLE
Add basic README.md

### PR DESCRIPTION
**Pull request by jasonharrison**
Adds a basic README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## USPSbot
+
+*USPSbot* a reddit bot that analyzes new posts submitted to [r/USPS](https://www.reddit.com/r/USPS).
```
Comment (flamecoals) [inline README.md]: Do USPSBot: or USPSBot is...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+If a new post appears to be related to tracking a package, *USPSbot* will search for a tracking number in the post's title and body.
```
Comment (flamecoals) [inline README.md]: How does it know if it is related to a tracking package?

Comment (flamecoals) [reply README.md]: Sounds good

Comment (jasonharrison) [reply README.md]: We must have been tired -- we both missed a typo:

> *USPSbot* determines the likelihood that a post is about tracking a package by comparing **the words in used** in the post's title/body to words frequently found in prior tracking questions.

**the words in used** 🤦‍♂️

**Pull request merged**